config/locales/en.yml

```diff
@@ -807,6 +807,57 @@ en:
       heading: Add Housing Benefit details
       housing_benefit_value: Enter amount
       housing_benefit_frequency: Select frequency
+    housing_costs:
+      heading: Housing costs
+      heading_hint: 
+        - "Enter the housing costs for the home the client lives in."
+      heading_hint_partner: 
+        - "Enter the housing costs for the home the client lives in."
+        - "If the client and partner live separately, do not include the partner’s housing costs."
+      housing_payments:
+        label: Housing payments
+        hint_paragraph:
```

Comment:
I'd rename this and similar ones below to `hint_paragraphs` with an `s`, to indicate that it's not a single string but rather an array of strings